map/src/infoblock/components/wpt/WptDetails.jsx

```diff
@@ -0,0 +1,350 @@
+        if (wpt.type?.isFav) {
+            return 'se-fav-item-info-' + wpt.name;
+        }
+        if (wpt.type?.isPoi) {
+            return 'se-poi-infoblock-' + wpt.name;
+        }
+        return null;
+    }
+
+    function formatTime(time) {
+        const locale = locales[i18n.language] || locales.enUS;
+        return format(time, 'MMM dd, yyyy – HH:mm', { locale: locale }).replace(',', '');
+    }
+
+    const Header = () => {
+        return (
+            <AppBar position="static" className={headerStyles.appbar}>
+                <Toolbar className={headerStyles.toolbar}>
+                    <IconButton
+                        variant="contained"
+                        type="button"
+                        className={styles.closeIcon}
+                        onClick={() => closeDetails()}
+                    >
+                        {isDetails ? <BackIcon /> : <CloseIcon />}
+                    </IconButton>
+                </Toolbar>
+            </AppBar>
+        );
+    };
+
+    const WptIcon = () => {
+        return (
+            <div
+                style={{ display: 'flex' }}
+                dangerouslySetInnerHTML={{
+                    __html:
+                        changeIconSizeWpt(
+                            removeShadowFromIconWpt(
+                                MarkerOptions.getWptIcon(wpt, wpt?.color, wpt?.background, wpt?.icon).options.html
+                            ),
+                            ICON_IMG_SIZE,
+                            ICON_SHIELD_SIZE
+                        ) + '',
+                }}
+            />
+        );
+    };
+
+    const WptCategory = () => {
+        return (
+            <Box className={styles.wptCategory}>
+                <ListItemIcon
+                    style={{
+                        color:
+                            wpt.category &&
+                            FavoritesManager.getColorGroup({ gpxFile: wpt.file, groupName: wpt.category }),
+                        minWidth: 'auto',
+                    }}
+                >
+                    <Folder fontSize="small" />
+                </ListItemIcon>
+                <Box>
+                    <Typography className={styles.wptCategoryText} noWrap>
+                        {`${wpt.category ? wpt.category : 'Favorites'} (${wpt.file?.wpts?.length})`}
+                    </Typography>
+                </Box>
+            </Box>
+        );
+    };
+
+    const WptAddress = () => {
+        return (
+            <Box className={styles.wptCategory}>
+                <ListItemIcon style={{ minWidth: 'auto' }}>
+                    <LocationOn fontSize="small" />
+                </ListItemIcon>
+                <ListItemText>
+                    <Typography className={styles.wptCategoryText} noWrap>
+                        {wpt.address}
+                    </Typography>
+                </ListItemText>
+            </Box>
+        );
+    };
+
+    const WptLoc = ({ wpt, location }) => {
```

Comment:
Please avoid showing this section when there is 0 km distance (= the favorite was just centered on the map).